### PR TITLE
Log record context

### PR DESCRIPTION
**Pull request by aarb1**

### CODE REVIEW

**Comment by wraschke** [comment]
From Erik:

Sorry for the confusion on the checklist, this is 1 of 6 PRs that were affected by the rollout of checklists for openliberty. It's not quite ready yet and being turned off, so for this PR please ignore the 'yellow' status check below, but do be on the lookout for future changes to how PRs are created and managed.

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils.java [hunk 1]
```diff
@@ -220,11 +223,28 @@ else if (key.equals(LogFieldConstants.MESSAGE)) {
...
+                            isFirstField = isFirstField & !CollectorJsonHelpers.addToJSON(sb, extKey, extValue, false, true, false, isFirstField, isExtQuoteless);
+                        }
+                    }
+                }
             }
-        }
 
-        if (tags != null) {
-            addTagNameForVersion(sb).append(CollectorJsonHelpers.jsonifyTags(tags));
+            if (tags != null) {
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils.java]: this should be outside of the for loop.

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils1_1.java [hunk 2]
```diff
@@ -331,11 +334,28 @@ else if (key.equals(LogFieldConstants.MESSAGE)) {
...
+                            isFirstField = isFirstField & !CollectorJsonHelpers.addToJSON(sb, extKey, extValue, false, true, false, isFirstField, isExtQuoteless);
+                        }
+                    }
+                }
             }
-        }
 
-        if (tags != null) {
-            addTagNameForVersion(sb).append(CollectorJsonHelpers.jsonifyTags(tags));
+            if (tags != null) {
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils1_1.java]: this should be outside of the for loop.

File: dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/LogFieldConstants.java [hunk 3]
```diff
@@ -39,4 +39,6 @@
     public static final String DATEOFFIRSTOCCURENCE = "dateOfFirstOccurence";
     public static final String COUNT = "count";
     public static final String IBM_REQUESTSTARTTIME = "ibm_requestStartTime";
+    public static final String EXT_PREFIX = "ext_";
+    public static final String KVPL_NAME = "extensions";
```
Comment (donbourne) [inline dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/LogFieldConstants.java]: need better name -- I'd suggest EXTENSIONS_KVPL

**Comment by aarb1** [comment]
#build

**Comment by aarb1** [comment]
#build

**Comment by aarb1** [comment]
#build

**Comment by fmhwong** [comment]
@aarb1 Is this still active?

**Pull request closed without merging**